src/lib/vega-wallet/vega-wallet-service.ts

```diff
@@ -176,6 +177,29 @@ export class VegaWalletService implements IVegaWalletService {
     return semver.satisfies(version, MINIMUM_WALLET_VERSION);
   }
 
+  async getLastestVersion() {
```

Comment:
@dexturr @pgriscti I'm wondering if this should be in the wallet service at all? After all we aren't really interacting with the wallet here, but the github API 🤔

Reply:
I'm leaning towards it just living in the component it gets used in or perhaps in a file in lib